Description: Do not link against libvpx
 None of its' symbols are used.
Author: Sebastian Ramacher <sramacher@debian.org>
Last-Update: 2014-09-27

--- a/gtk/configure.ac
+++ b/gtk/configure.ac
@@ -158,7 +158,7 @@
 GHB_TOOLS_CFLAGS=`$BUILD_PKG_CONFIG --cflags glib-2.0 gdk-pixbuf-2.0`
 GHB_TOOLS_LIBS=`$BUILD_PKG_CONFIG --libs glib-2.0 gdk-pixbuf-2.0`
 
-HB_LIBS="-lhandbrake -lavresample -lavformat -lavcodec -lavutil -ldvdnav -ldvdread -lmp3lame -lvorbis -lvorbisenc -logg -lsamplerate -lx264 -lswscale -ltheoraenc -ltheoradec -lvpx -lz -lbz2 -lbluray -lass -lfontconfig -lfreetype -lxml2"
+HB_LIBS="-lhandbrake -lavresample -lavformat -lavcodec -lavutil -ldvdnav -ldvdread -lmp3lame -lvorbis -lvorbisenc -logg -lsamplerate -lx264 -lswscale -ltheoraenc -ltheoradec -lz -lbz2 -lbluray -lass -lfontconfig -lfreetype -lxml2"
 
 case $host in
   *-*-mingw*)
--- a/libhb/module.defs
+++ b/libhb/module.defs
@@ -107,7 +107,7 @@
 LIBHB.dll.libs = $(foreach n, \
         ass avcodec avformat avutil avresample dvdnav dvdread \
         fontconfig freetype mp3lame \
-        ogg samplerate swscale vpx theora vorbis vorbisenc x264 x265 xml2 bluray, \
+        ogg samplerate swscale theora vorbis vorbisenc x264 x265 xml2 bluray, \
         $(CONTRIB.build/)lib/lib$(n).a )
 
 ifeq (1,$(FEATURE.fdk_aac))
--- a/test/module.defs
+++ b/test/module.defs
@@ -16,7 +16,7 @@
 TEST.GCC.l = \
         ass avresample avformat avcodec avutil mp3lame dvdnav dvdread \
         fontconfig ogg \
-        samplerate swscale vpx theoraenc theoradec vorbis vorbisenc x264 x265 \
+        samplerate swscale theoraenc theoradec vorbis vorbisenc x264 x265 \
         bluray freetype xml2 bz2 z
 
 ifeq (1,$(FEATURE.qsv))
